__integrations__/index.test.ts

```diff
@@ -505,6 +508,78 @@ describe("Create Advanced Message", () => {
     }
   );
 
+  it("should return the message without Read status, if user is allowed to read it", async () => {
```

Comment:
It seems that there is an incoherence in title. If it does not return read status it should be because user is NOT allowed to read it. Am I wrong?